### PR TITLE
Add smardex on polygon to dex.trades

### PR DESCRIPTION
**Pull request by blanchemaria6**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above
---
### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** Add smardex on polygon to dex.trades

### CODE REVIEW

**Comment by blanchemaria6** [comment]
I have read the CLA Document and I hereby sign the CLA

File: seeds/_sector/dex/smardex_polygon_base_trades_seed.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+polygon,smardex,1,2024-06-11,0x885b2833498c911aca89a8c0219686583538c75706795b25c478cba709756418,476,0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6,0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270,58014407,3290,3503000228043981967
+polygon,smardex,1,2024-06-11,0xddc565236cbf833c0e56404002cd93ee2c2435f642c0ea1c00d0d48046eb3e84,170,0x111111517e4929d3dcbdfa7cce55d30d4b6bc4d6,0x2791bca1f2de4661ed88a30c99a7a9449aa84174,58015096,226786000000000000,581594
```
Comment (Hosuke) [inline seeds/_sector/dex/smardex_polygon_base_trades_seed.csv:3]: Hi @blanchemaria6, please verify seed records, seems token_bought and token_sold are flipped:
https://dune.com/queries/3821590

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: seeds/_sector/dex/smardex_polygon_base_trades_seed.csv (hunk 1 above)
Comment (blanchemaria6) [reply seeds/_sector/dex/smardex_polygon_base_trades_seed.csv]: I fixed the seed file and it works now @Hosuke

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @blanchemaria6 

[Sample smardex_polygon_base_trades](https://dune.com/queries/3825733)

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**